### DISCUSSION TITLE
I can't log back into my account after losing my email

### DISCUSSION THREAD

**Discussion by mirabble** [community/community#121611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

so for some unknown reason my google account got suspended and it seems like it is not coming back. since then I have lost access to my main github account. is there any way to change it's primary email or at least delete it ? I am still logged in with github desktop. could that help ?

**Comment by ebndev** [comment]
Hi @GoodErnest64,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
